### ISSUE TITLE
Multi-Scale Structural Similarity

### ISSUE THREAD

**Issue by jorge-pessoa** [Po-Hsun-Su/pytorch-ssim#9]
Hello, 

This SSIM implementation is a great addiction to the differentiable functions of PyTorch. However it is lacking the Multi-Scale Structural Similarity, a more commonly used metric in image assessment. I think it would be beneficial and relatively simple to implement and add this metric to this project.

I wrote a basic working MS-SSIM implementation based on SSIM from this project and using the Tensorflow implementation as reference. Would you consider adding the MS-SSIM metric if a pull request was sent?

Thank you,

**Comment by chaos5958** [comment]
Dear jorge-pessoa,

I also need MS-SSIM, how is your progress?
Could you share your version of MS-SSIM.

In addtion, I wonder whether there is GPU version of MS-SSIM regardless of using pytorch. (BUT SHOULD RUN ON PYTHON)

Thanks,
Hyunho
